modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/PodcastItem.kt

```diff
@@ -32,14 +32,16 @@ fun PodcastItem(
     showSubscribed: Boolean = false,
     showPlusIfUnsubscribed: Boolean = false,
     showDivider: Boolean = true,
+    onSubscribeClick: (() -> Unit)? = null,
```

Comment:
If we renamed this parameter to be `onPlusClick` the connection between this parameter and the `showPlusIfUnsubscribed` parameter would be more obvious to me. This might just be my personal preference though, so feel free to leave as-is if you prefer.

Reply:
Good point, I renamed it to `onPlusClick` - 83727aa